components/multi-platform-controller/staging-downstream/host-config.yaml

```diff
@@ -406,7 +406,7 @@ data:
 
   dynamic.linux-root-amd64.type: aws
   dynamic.linux-root-amd64.region: us-east-1
-  dynamic.linux-root-amd64.ami: ami-026ebd4cfe2c043b2
```

Comment:
I think there is an particular reason why the `root`-able machines have an different AMI... (I don't remember it, through...)